javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,35 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+  if(class07Students.length<6 || studentName ==='Queen'){
+      class07Students.push(studentName);
+      console.log(studentName);
+      return;
+  }else if(class07Students.includes(studentName)){
+      console.log `Student ${studentName} is already in the class`;
+      return;
+  }else{
+      console.log('Canot add more students to the clas 07');
+      return;
+  }
+}
```

Comment:
Try add `Queen` twice :wink:

Reply:
I see you added the Queen twice :+1:  BUT it was supposed to make you pause and think:
Why is it okay to add the Queen twice (or event 3 or 4 times), when a normal student can only be added a single time?

I would say you have a bug here :wink: 

Neither a normal student nor the Queen should be able to be added more than a single time.